### PR TITLE
handle RPL_UMODEIS

### PR DESCRIPTION
**Pull request by brunnre8**

### CODE REVIEW

File: src/plugins/irc-events/mode.js [hunk 1]
```diff
@@ -39,6 +39,17 @@ module.exports = function (irc, network) {
 		targetChan.pushMessage(client, msg);
 	});
 
+	irc.on("user info", function (data) {
+		const serverChan = network.channels[0];
```
Comment (MaxLeiter) [inline src/plugins/irc-events/mode.js:43]: do we want to show this in the open channel if it exists?

Comment (brunnre8) [reply src/plugins/irc-events/mode.js:43]: If you prefer?
I wasn't quite sure what to do with it, so I shoved it into the server buffer

Comment (MaxLeiter) [reply src/plugins/irc-events/mode.js:43]: Yeah, I think this should go in `activeChannel`, similar to notices targeted at the user.

**Comment by brunnre8** [comment]
Added the corresponding showInActive flag, should be good now.

**Review by MaxLeiter — APPROVED** [review]

**Review by itsjohncs — APPROVED** [review]
Did a quick test, enough to see the message (and confirmed it'll appear in the active channel). LGTM

**Pull request merged**